src/publish/confluence/confluence-helper.ts

```diff
@@ -673,3 +689,23 @@ export const getAttachmentsDirectory = (
 
   return result;
 };
+
+export const getAttachmentsDirectory = (
```

Comment:
Is this function actually used somewhere (didn't see it but it looks a little like a work in progress)

Reply:
You zoned right in on it.  Yep it has been removed- dead code.